constants.js

```diff
@@ -0,0 +1 @@
+export const Limit = 9;
```

Comment:
convention: use UPPERCASE

Reply:
resolved in PR 7 .Do i need to fix it here?

Reply:
Resolved